plugins/uberdev/skills/testers-pipeline/aggregate.py

```
@@ -0,0 +1,88 @@
+            dispositions.append(d)
+
+    out = {
+        "schema_version": 1,
+        "wave": args.wave,
+        "run_id": args.run_id,
+        "findings": findings,
+        "cross_refs": cross_refs_in,
+        "follow_ups_for_next_wave": follow_ups,
+        "dispositions": dispositions,
+    }
+    yaml.safe_dump(out, open(args.out, "w"), default_flow_style=False, sort_keys=False)
```

Ruff (0.15.13)

[warning] Use a context manager for opening files

(SIM115)